edi_punchout/views/edi_punchout_account.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_edi_punchout_account_form" model="ir.ui.view">
+    <field name="model">edi.punchout.account</field>
+    <field name="arch" type="xml">
+        <form>
+            <sheet>
+                <group>
+                    <group>
+                        <field name="partner_id" />
+                        <field name="company_id" groups="base.group_multi_company" />
+                        <field name="group_ids" widget="many2many_tags" />
+                    </group>
+                    <group>
+                        <field name="protocol" />
+                        <field
+                                name="oci_version"
+                                attrs="{'invisible': [('protocol', '!=', 'oci')], 'required': [('protocol', '=', 'oci')]}"
+                            />
+                        <field
+                                name="ids_version"
+                                attrs="{'invisible': [('protocol', '!=', 'ids')], 'required': [('protocol', '=', 'ids')]}"
+                            />
+                        <field name="catalog_url" />
+                        <field
+                                name="oci_custom_url_parameters"
+                                attrs="{'invisible': [('protocol', '!=', 'oci')], 'required': [('protocol', '=', 'oci')]}"
+                            />
+                        <field
+                                name="ids_kndnr"
+                                attrs="{'invisible': [('protocol', '!=', 'ids')]}"
+                            />
+                        <field
+                                name="ids_name_kunde"
+                                attrs="{'invisible': [('protocol', '!=', 'ids')]}"
+                            />
+                        <field
+                                name="ids_pw_kunde"
```

Comment:
```suggestion
                                name="ids_pw_kunde" password="1"
```

Reply:
seems sensible. Note however I'm not on the project using this any more, so I'll be slow in updating this PR